content/sensu-go/6.3/observability-pipeline/observe-schedule/rule-templates.md

```diff
@@ -0,0 +1,758 @@
+Both service components can make use of the same rule template at the threshold that makes sense for that component.
+
+Service components can reference more than one rule template.
+Sensu evaluates each rule separately, and each rule produces its own event as output.
+
+## Rule template example
+
+This example rule template creates an event when the percentage of events with the given status exceed the given threshold:
+
+{{< language-toggle >}}
+
+{{< code yml >}}
+---
+type: RuleTemplate
+api_version: bsm/v1
+metadata:
+  name: status-threshold
+  namespace: default
+spec:
+  description: Creates an event when the percentage of events with the given status exceed the given threshold
+  arguments:
+    required:
+      - threshold
+    properties:
+      status:
+        type: string
+        default: non-zero
+        enum:
+          - non-zero
+          - warning
+          - critical
+          - unknown
+      threshold:
+        type: number
+        description: Numeric value that triggers an event when surpassed
+  eval: |
+    var statusMap = {
+      "non-zero": 1,
+      "warning": 1,
+      "critical": 2,
+    };
+    function main(args) {
+      var total = sensu.events.count();
+      var num = sensu.events.count(args.status);
+      if (num / total <= args.threshold) {
+        return;
+      }
+      return event.status = statusMap[args.status],
+      });
+    }
+{{< /code >}}
+
+{{< code json >}}
+{
+  "type": "RuleTemplate",
+  "api_version": "bsm/v1",
+  "metadata": {
+    "name": "status-threshold",
+    "namespace": "default"
+  },
+  "spec": {
+    "description": "Creates an event when the percentage of events with the given status exceed the given threshold",
+    "arguments": {
+      "required": [
+        "threshold"
+      ],
+      "properties": {
+        "status": {
+          "type": "string",
+          "default": "non-zero",
+          "enum": [
+            "non-zero",
+            "warning",
+            "critical",
+            "unknown"
+          ]
+        },
+        "threshold": {
+          "type": "number",
+          "description": "Numeric value that triggers an event when surpassed"
+        }
+      }
+    },
+    "eval": "var statusMap = {\n  \"non-zero\": 1,\n  \"warning\": 1,\n  \"critical\": 2,\n};\nfunction main(args) {\n  var total = sensu.events.count();\n  var num = sensu.events.count(args.status);\n  if (num / total <= args.threshold) {\n    return;\n  }\n  return event.status = statusMap[args.status],\n  });\n}"
+  }
+}
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+## Built-in rule templates
+
+Sensu's business service monitoring (BSM) includes built-in rule templates that address common BSM use cases.
+These built-in rule templates are ready to use with your service components.
+Reference the rule template name in the `rules.template` field and configure the arguments in the `rules.template.arguments` object in your service component resource definitions.
+
```

Comment:
Are there more built-in rule templates -- if so, where can I find them?

Reply:
As it stands, we only have 1: `aggregate-rule-template`.